### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by abhi18av** [brynbellomy/dash-docset-generator#5]
Hi @brynbellomy  

I was trying to generate the `selenium-java` docs from the website here http://seleniumhq.github.io/selenium/docs/api/java/index.html

And I've downloaded the website using `wget -r url` command.

But while trying to install this npm package, I get the following error 

```sh
❯ npm install -g dash-docset-generator
npm ERR! Error while executing:
npm ERR! /usr/local/bin/git ls-remote -h -t git://github.com/brynbellomy/command-router.git
npm ERR!
npm ERR! fatal: remote error:
npm ERR!   Repository not found.
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/eklavya/.npm/_logs/2017-09-23T16_43_45_678Z-debug.log

apply_bot/_resources git/master*  8s
❯ node --version
v8.1.3


```

**Comment by Olliebrown** [comment]
I have this problem too.  The package appears to depend on several other packages from Bryn and this one (command-router) is gone.  It tries to retrieve ` git://github.com/brynbellomy/command-router.git` and then reports the error `Repository not found.`

I have looked for this repository manually (Under Bryn's repos) and it is not there so it must have been removed for some reason.  The same error occurs with Yarn and it also occurs with the pre-packaged release version as this is merely the typescript recompiled to run as a js node application and it still requires the node package dependencies (and they are not there in the release zip file).

Basically, this project is dead unless the command-router repo comes back or the project is re-written to not depend on this particular package.  <sad panda>
